canonical/canonical.com
# improve Google JobPosting

## Done

- Add missing [applicantLocationRequirements](https://developers.google.com/search/docs/appearance/structured-data/job-posting#applicant-location-requirements) field: the location of the candidate in case the job is remote
- Set a unified `rel=canonical` link for both routes `/careers/:id` and `/careers/:id/:title`: in case of having multiple URLs that point to the same page it's recommended to specify `link rel=canonical` to one of the URLs as the source of truth

## QA

- Go to https://search.google.com/test/rich-results
- Test a job post from the current website: https://canonical.com/careers/2804965
- Test a job post from demo: https://canonical-com-672.demos.haus/careers/2804965
- Make sure that there is no errors in the demo version compared to prod
